### DISCUSSION TITLE
Sample automation

### DISCUSSION THREAD

**Discussion by semtxxx** [SynthstromAudible/DelugeFirmware#3093] [Ideas]
It would be wonderful if sample automation via lfo or parameter lock  is possible like 
Reverse 
Pitch 
Start End point 
Maybe even sample flipping to choose from folders 
Some kind of granular synthesis 
Dunno if anything of this is possible , i know some modular units that can do that but don't know if the delly could handle or do that

**Comment by CalitoH** [comment]
I agree so much, this would be so great to have those features! I used to have the digitakt and those sample manipulations are the thing I miss the most.
